src/Oracles/Config/Setting.hs

```diff
@@ -51,6 +52,16 @@ data Setting = BuildArch
              | GmpLibDir
              | IconvIncludeDir
              | IconvLibDir
+             | Prefix
```

Comment:
Same here: let's keep `Install` prefix for all install-related settings to keep the code a bit easier to understand.